templates/part.spreedmeeting.php

```diff
@@ -28,11 +28,11 @@
 	// TODO(leon): Include link to app store (https://apps.nextcloud.com/apps/spreed)
 } else {?>
 
-<label>
-	<input
-		type="checkbox" class="spreedmeetingcreatecheckbox event-checkbox"
-		ng-model="properties.doScheduleMeeting" />
-	<?php p($l->t('Schedule Spreed Meeting for this event'));?>
+<input type="checkbox" name="spreedmeetingcreatecheckbox"
```

Comment:
Any reason the `label` no longer contains the `input` field? This requires us to introduce the `id` tag on `input`.

Reply:
most of the other code looks like that!